packs/github/actions/lib/base.py

```diff
@@ -8,19 +8,20 @@
     'BaseGithubAction'
 ]
 
-BASE_URL = 'https://github.com'
-
 
 class BaseGithubAction(Action):
     def __init__(self, config):
         super(BaseGithubAction, self).__init__(config=config)
         token = self.config.get('token', None)
         token = token or None
-        self._client = Github(token)
+        base_url = self.config.get('BASE_URL', None)
+        base_url = base_url or None
+        self._client = Github(token, base_url)
 
     def _web_session(self):
         '''Returns a requests session to scrape off the web'''
-        login_url = BASE_URL + '/login'
```

Comment:
Change `BASE_URL` to `WEB_URL`, don't use the url from config: it's a web url.
Same on line 43.